### PR TITLE
Copy edit FAQs

### PR DESCRIPTION
**Pull request by charislam**
# Description

[Short summary of why you created this PR]

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

File: timescaledb/overview/faq/faq-postgres.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ Read our TimescaleDB-PostgreSQL benchmarks:
 To summarize, TimescaleDB offers:
 * Ease-of-use: TimescaleDB is far easier to use because creating partitions (or what we call
 "chunks") is automatically performed for the user. All of the complexity of automatic
-partitioning is abstracted away behind a "hypertable", which users interact with just as
+partitioning is abstracted away behind a "hypertable," which users interact with just as
 they would with a PostgreSQL table.
 * Much higher ingest scale: TimescaleDB sees throughput more than 20X that of
-PostgreSQL once tables reach moderate size (e.g., 10s of millions of rows).
+PostgreSQL once tables reach moderate size (for example, 10s of millions of rows).
```
Comment (charislam) [inline timescaledb/overview/faq/faq-postgres.md:14]:
```suggestion
PostgreSQL once tables reach moderate size (for example, 10&nbsp;s of millions of rows).
```

Comment (charislam) [inline timescaledb/overview/faq/faq-postgres.md:14]:
```suggestion
<!-- vale Google.Units = NO -->
PostgreSQL once tables reach moderate size (for example, 10s of millions of rows).
<!-- vale Google.Units = YES -->
```

File: timescaledb/overview/faq/faq-postgres.md [hunk 2]
```diff
@@ -8,10 +8,13 @@ Read our TimescaleDB-PostgreSQL benchmarks:
 To summarize, TimescaleDB offers:
 * Ease-of-use: TimescaleDB is far easier to use because creating partitions (or what we call
 "chunks") is automatically performed for the user. All of the complexity of automatic
-partitioning is abstracted away behind a "hypertable", which users interact with just as
+partitioning is abstracted away behind a "hypertable," which users interact with just as
```
Comment (Loquacity) [inline timescaledb/overview/faq/faq-postgres.md:11]:
```suggestion
partitioning is abstracted away behind a hypertable, which users interact with just as
```

File: timescaledb/overview/faq/faq-postgres.md [hunk 3]
```diff
@@ -8,10 +8,13 @@ Read our TimescaleDB-PostgreSQL benchmarks:
...
 they would with a PostgreSQL table.
-* Much higher ingest scale: TimescaleDB sees throughput more than 20X that of
-PostgreSQL once tables reach moderate size (e.g., 10s of millions of rows).
+* Much higher ingest scale: TimescaleDB sees throughput more than 20X that of
```
Comment (Loquacity) [inline timescaledb/overview/faq/faq-postgres.md:13]:
```suggestion
* Much higher ingest scale: TimescaleDB sees throughput more than 20 times that of 
```

File: timescaledb/overview/faq/faq-postgres.md [hunk 4]
```diff
@@ -8,10 +8,13 @@ Read our TimescaleDB-PostgreSQL benchmarks:
...
 * Ease-of-use: TimescaleDB is far easier to use because creating partitions (or what we call
 "chunks") is automatically performed for the user. All of the complexity of automatic
-partitioning is abstracted away behind a "hypertable", which users interact with just as
+partitioning is abstracted away behind a "hypertable," which users interact with just as
 they would with a PostgreSQL table.
-* Much higher ingest scale: TimescaleDB sees throughput more than 20X that of
-PostgreSQL once tables reach moderate size (e.g., 10s of millions of rows).
+* Much higher ingest scale: TimescaleDB sees throughput more than 20X that of 
+<!-- vale Google.Units = NO -->
+PostgreSQL once tables reach moderate size (for example, 10s of millions of
```
Comment (Loquacity) [inline timescaledb/overview/faq/faq-postgres.md:15]:
```suggestion
PostgreSQL once tables reach moderate size (for example, tens of millions of
```

File: timescaledb/overview/faq/faq-products.md [hunk 5]
```diff
@@ -169,8 +169,8 @@ and inherits its support for varied datatypes and indexes (B-tree, hash, range,
 * If support for geospatial data is desirable. Data stored in TimescaleDB can leverage PostGIS's
 geometric datatypes, indexes, and queries.
 * If you want greater optionality when it comes to using third-party tools. TimescaleDB supports
-anything that speaks SQL (i.e., the entire PostgreSQL ecosystem).
-* If you already use and like PostgreSQL, and don't want to have to "give it up" and move to a
+anything that speaks SQL (in other words, the entire PostgreSQL ecosystem).
```
Comment (Loquacity) [inline timescaledb/overview/faq/faq-products.md:172]:
```suggestion
anything that speaks SQL, including the entire PostgreSQL ecosystem.
```

File: timescaledb/overview/faq/faq-products.md [hunk 6]
```diff
@@ -169,8 +169,8 @@ and inherits its support for varied datatypes and indexes (B-tree, hash, range,
 * If support for geospatial data is desirable. Data stored in TimescaleDB can leverage PostGIS's
 geometric datatypes, indexes, and queries.
 * If you want greater optionality when it comes to using third-party tools. TimescaleDB supports
```
Comment (Loquacity) [inline timescaledb/overview/faq/faq-products.md:171]:
```suggestion
* If you want more options when it comes to using third-party tools. TimescaleDB supports
```

File: timescaledb/overview/faq/faq-products.md [hunk 7]
```diff
@@ -169,8 +169,8 @@ and inherits its support for varied datatypes and indexes (B-tree, hash, range,
...
-anything that speaks SQL (i.e., the entire PostgreSQL ecosystem).
-* If you already use and like PostgreSQL, and don't want to have to "give it up" and move to a
+anything that speaks SQL (in other words, the entire PostgreSQL ecosystem).
+* If you already use and like PostgreSQL, and don't want to "give it up" and move to a
```
Comment (Loquacity) [inline timescaledb/overview/faq/faq-products.md:173]:
```suggestion
* If you already use and like PostgreSQL, and don't want to give it up and move to a
```

File: timescaledb/overview/faq/faq-products.md [hunk 8]
```diff
@@ -179,20 +179,21 @@ system due to scaling concerns or issues. We provide support for the migration b
 For this scenario, in-memory or column-oriented databases are designed for
 key-value storage with fast lookup and a relational database may not be ideal.
 However, these systems clearly do not scale to large data volumes and cannot
-perform well for more complex queries (whereas relational databases
+perform well for more complex queries (where relational databases
 like TimescaleDB and PostgreSQL are better suited).
```
Comment (Loquacity) [inline timescaledb/overview/faq/faq-products.md:183]:
```suggestion
perform well for more complex queries, where relational databases
like TimescaleDB and PostgreSQL are better suited.
```

File: timescaledb/overview/faq/faq-using-timescaledb.md [hunk 9]
```diff
@@ -54,7 +54,7 @@ as writes (inserts) to the latest time intervals, not as updates to existing rec
 * Workloads have a natural partitioning across both time and space.
 
 TimescaleDB leverages these properties by automatically partitioning data into
-two-dimensional "chunks" (i.e., smaller PostgreSQL tables), performing
+two-dimensional "chunks" (smaller PostgreSQL tables), performing
```
Comment (Loquacity) [inline timescaledb/overview/faq/faq-using-timescaledb.md:57]:
```suggestion
two-dimensional chunks that operate like smaller PostgreSQL tables, performing
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Review by Loquacity — APPROVED** [review]

**Comment by Loquacity** [comment]
Don't forget to update the PR description 😉

**Pull request merged**